map/map_loader/src/pointcloud_map_loader/pointcloud_map_loader_node.cpp

```diff
@@ -89,8 +89,25 @@ std::map<std::string, PCDFileMetadata> PointCloudMapLoaderNode::getPCDMetadata(
   const std::string & pcd_metadata_path, const std::vector<std::string> & pcd_paths) const
 {
   if (fs::exists(pcd_metadata_path)) {
+    std::set<std::string> missing_pcd_names;
     auto pcd_metadata_dict = loadPCDMetadata(pcd_metadata_path);
-    pcd_metadata_dict = replaceWithAbsolutePath(pcd_metadata_dict, pcd_paths);
+
+    pcd_metadata_dict = replaceWithAbsolutePath(pcd_metadata_dict, pcd_paths, missing_pcd_names);
+
+    // Warning if some segments are missing
+    if (!missing_pcd_names.empty()) {
+      std::ostringstream oss;
+
+      oss << "The following segment(s) are missing from the input PCDs: " << std::endl;
+
+      for (auto & fname : missing_pcd_names) {
+        oss << fname << std::endl;
+      }
+
+      RCLCPP_ERROR_STREAM(get_logger(), oss.str());
+      throw std::runtime_error("Missing PCD segments. Exiting map loader...\n");
```

Comment:
The redundant line breaks make the error message strange. Please fix it :pray:

![image](https://github.com/autowarefoundation/autoware.universe/assets/24854875/0855800d-426a-4d0f-a6d8-f196ec4fd6f6)


How about replacing with the following
```cpp
  for (auto & fname : missing_pcd_names) {
      RCLCPP_ERROR_STREAM(get_logger(), fname);
      }
```